openssl-sys/Cargo.toml

```diff
@@ -15,9 +15,11 @@ build = "build/main.rs"
 
 [features]
 vendored = ['openssl-src']
+boringssl = ['bssl-sys']
```

Comment:
Can we rename this feature to `unstable_boringssl` or something like that for now? I just want to have some time for external consumers to pick this up before committing to the bssl-sys patching approach.